opossum-tool/OpossumUI
# feat: merge progress data workers into signal worker

### Summary of changes

- merge progress data workers into signals worker to save on memory (they mostly need the same input data)
- delete any remnants of separate workers and their context providers

### Context and reason for change

#2384 

### How can the changes be tested

As you're switching between resources in audit view and check that the progress bars are still working as expected, look at the memory tab in the devtools to see that the memory footprint has been reduced compared to the base branch.
